dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -198,3 +198,9 @@ http.proxyredirect.httpsPort.desc=The (secure) port to redirect to. Incoming HTT
 
 http.autoDecompression=Auto decompression
 http.autoDecompression.desc=Specifies whether the HTTP Channel automatically decompresses incoming request body data.
+
+http.decompressionRatioLimit=Decompression ratio limit
+http.decompressionRatioLimit.desc=Specifies the maximum ratio of decompressed to compressed request body payload. This ratio is verified as the body is read by the HTTP channel, and if the decompressionTolerance is reached, the request will be terminated.
+
+http.decompressionTolerance=Decompression tolerance
+http.decompressionTolerance.desc=Specifies the maximum number of times the HTTP channel can hit the decompressionRatioLimit as the request's body is decompressed before the request is terminated.
```

Comment:
Suggested change: "Specifies the maximum number of times the HTTP channel can reach the decompressionRatioLimit before the request to decompress the request body is ended."

At minimum, change "is terminated" to "is ended" and remove the apostrophe in "request's."